go.mod

```diff
@@ -35,5 +35,4 @@ require (
 	k8s.io/apimachinery v0.18.2
 	k8s.io/client-go v12.0.0+incompatible
 	sigs.k8s.io/controller-runtime v0.6.0
-	sigs.k8s.io/structured-merge-diff v0.0.0-20190525122527-15d366b2352e // indirect
```

Comment:
Why was this removed?

Reply:
This was removed automatically, I am unsure why that happened

Reply:
What did you run that removed this?